components/Cases/CaseRecap/CaseRecap.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import Summary from 'components/Summary/Summary';
+import { formatDate } from 'utils/date';
+import { useCaseByResident } from 'utils/api/cases';
+import * as form from 'data/forms';
+
+const CaseRecap = ({ personId, recordId }) => {
+  const { data: record, error: recordError } = useCaseByResident(
+    personId,
+    recordId
+  );
+
+  const recordData =
+    record && record.caseFormData && record.caseFormData.form_name_overall;
+  const fileData = form[recordData];
+
+  if (recordError || (recordData && !fileData)) {
+    return <ErrorMessage />;
+  }
+  if (!record || !fileData) {
+    return <Spinner />;
+  }
+  return (
+    <>
+      <h3 className="govuk-heading-m govuk-custom-text-color govuk-!-margin-bottom-0">
```

Comment:
I guess we want to use the same styling used for the subheaders, like:

<img width="330" alt="Screenshot 2021-03-08 at 10 58 20" src="https://user-images.githubusercontent.com/435399/110312631-9f419800-8005-11eb-8a71-4d0307571323.png">